fakestorage/object.go

```diff
@@ -124,11 +142,24 @@ func fromBackendObjects(objects []backend.Object) []Object {
 			Crc32c:          o.Crc32c,
 			Md5Hash:         o.Md5Hash,
 			ACL:             o.ACL,
+			Created:         convertTimeWithoutError(o.Created),
+			Deleted:         convertTimeWithoutError(o.Deleted),
+			Updated:         convertTimeWithoutError(o.Updated),
+			Generation:      o.Generation,
 		})
 	}
 	return backendObjects
 }
 
+// https://github.com/googleapis/google-cloud-go/blob/2f857649c55302802e95b96119dd05032a61c87a/storage/storage.go#L1023
+func convertTimeWithoutError(t string) time.Time {
+	var r time.Time
+	if t != "" {
+		r, _ = time.Parse(time.RFC3339, t)
+	}
+	return r
```

Comment:
Can simplify this function:

```go
r, _ := time.Parse(time.RFC3339, t)
return r
```

Reply:
I used exactly the same function google is also using, probably not to couple the error response of time.Parse to the api response... but yeah, its quite natural that, in case of errors, they return a null value that is fine for us, so changing!